Dockerfile

```diff
@@ -29,13 +29,13 @@ RUN apt-get update \
         libreadline-dev \
         libsqlite3-dev \
         libtbb-dev \
-        lld-9 \
+        lld \
```

Comment:
Did you already push a new Docker image?

Reply:
Yes